doc/user/pathd.rst

```diff
@@ -6,11 +6,164 @@ PATH
+(adjacency changes,...).
+Based in the support of IS-IS or OSPF we can activate TED support what will
+allow pathd to resolve MPLS based in different types of segments
+(:rfc: `draft-ietf-spring-segment-routing-policy-07`). The supported types are
+Type C (prefix and local interface), Type E (prefix and algorithm),
+Type F (a pair of IP's).
+So the configuration would change to this
+
+.. code-block:: frr
+
+  segment-routing
+   traffic-eng
+    mpls-te on
+    mpls-te import ospfv2
+    segment-list SL1
+     index 10  nai prefix 10.1.2.1/32 iface 1
+     index 20  nai adjacency 10.1.20.1 10.1.20.2
+    !
+    policy color 1 endpoint 4.4.4.4
+     name example1
+     binding-sid 1111
+     candidate-path preference 100 name CP1 explicit segment-list SL1
+
+
+In this case no MPLS are provided but the pathd TED support will resolve the
+configuration provided to corresponding MPLS labels.
+
+.. note::
+	Please notes the ``mpls-te`` configuration added that activate the
+	support and points to ``ospfv2`` so
+	the ospfv2 (:ref:`ospf-traffic-engineering`) daemon must be also 
+	running and configure to export TED information.
+
+.. note::
+	It would be the same for isis (:ref:`traffic-engineering`) but in the
+	moment of writting it's not fully tested.
+
+Dynamic Segment Lists
+---------------------
+
+One of the useful options to configure is the creation of policies with
+the dynamic option. In this case based in a given endpoint the SL will be
+,first calculated, and then sended by means of PCEP protocol by the configured
+PCE.
+
+.. code-block:: frr
+
+   traffic-eng
+    !
+    pcep
+     !
+     pce PCE1
+      address ip 10.10.10.10
+     !
+     pcc
+      peer PCE1 precedence 10
+    !
+    policy color 1 endpoint 4.4.4.4
+     name example
+     binding-sid 1111
+     candidate-path preference 100 name CP2 dynamic
+
+.. note::
+	Please notes the config for the remote pce to connect to and so pathd
+	is a PCEP client by adding the pcc config.
+
+.. note::
+	If is active the TED support that feature will validate data from PCE, 
+	so in a SL from PCEP/PCE the IP and MPLS will be validated against 
+	local TED obtained/built from igp configured in that case.
+
+Pce Initiated
+-------------
+
+We can step forward in the use of our controller not only by asking to
+calculate paths to an endpoint but also to create the whole policies in the
+controller and obtain those by means of the PCEP protocol.
+
+
+.. code-block:: frr
+
+   traffic-eng
+    !
+    pcep
+     !
+     pce PCE1
+      address ip 10.10.10.10
+      pce-initiated
+     !
+     pce PCE2
+      address ip 9.9.9.9
+      pce-initiated
+     !
+     pcc
+      peer PCE1 precedence 10
+      peer PCE2 precedence 20
+    !
+
+.. note::
+	Now there is not policies in the config as they will obtain from pce.
+	Please check command :clicmd:`show sr-te policy` in ``vtysh`` to see
+	the obtained policies another interesting command is
```

Comment:
Seems like this should be another sentense? `Another interesting...`

Reply:
Thanks for your comments ... will be covered in the next force commit